ember-core/shared/src/main/scala/org/http4s/ember/core/Util.scala

```diff
@@ -36,7 +36,7 @@ private[ember] object Util extends UtilPlatform {
   private[this] val closeCi = ci"close"
   private[this] val keepAliveCi = ci"keep-alive"
   private[this] val connectionCi = ci"connection"
-  private[this] val close = Connection(NonEmptyList.of(closeCi))
+  val close: Connection = Connection(NonEmptyList.of(closeCi))
```

Comment:
Maybe some private context isn't that bad?
```suggestion
  private[http4s] val close: Connection = Connection(NonEmptyList.of(closeCi))
```